disdrodb/tests/test_standards.py

```diff
@@ -0,0 +1,120 @@
+import yaml
+from disdrodb.L0 import standards
+
+# The yaml files are tested in the test_config_files.py file
+
+"""
+
+def test_read_config_yml():

```

Comment:
I would simply check that all these functions, for each of the sensor name in configs, returns an object of dictionary, list or whatever expected dtype !

Reply:
ok I create a new issue for that